### ISSUE TITLE
.people() ignores name particles

### ISSUE THREAD

**Issue by CalvinWu4** [spencermountain/compromise#794]
"de Marco" is being parsed as "Marco" when I do people().out()

**Comment by spencermountain** [comment]
hi Calvin, good catch. Got a fix on dev, will be part of the next release.
cheers

**Comment by CalvinWu4** [comment]
You should also check for the Spanish "y" if you haven't

**Comment by CalvinWu4** [comment]
Maybe even add an option to remove lowercase last name particles in .people()

**Comment by spencermountain** [comment]
sure, can you give an example?

**Comment by CalvinWu4** [comment]
Julien de Bussy might go as Julien Bussy for example

Or you can just link me to the particles caught in the code, so I can check for it manually

**Comment by spencermountain** [comment]
ah, good idea.
Yeah, I love the idea of parsing a person's name into components, and adding methods to normalize them or whatever.

Earlier versions of compromise had methods on Person class that parsed-out `honoric, firstname, initial, lastname` and stuff
you can see the Person class is [really-sad](https://github.com/spencermountain/compromise/blob/master/src/Subset/People.js) right now. It's a great idea, and prs are welcome. I'd be happy to help.

removing a particle would probably look something like this:
```js
doc.people().match('#FirstName [(van|von|de|du)] #LastName', 0).remove()
``

ideally calling `.people().normalize()` would do something smart, so `Mrs Hilary R. Clinton` can be cleaned-up for easier text-processing, without doing anything too culturally insensitive. In Poland, last-name indicates gender, and stuff like that - that would be neat to turn into a person plugin.

**Comment by spencermountain** [comment]
should be fixed on 13.9 now
cheers
